Library/NumberFormatterTests.swift

```diff
@@ -0,0 +1,150 @@
+@testable import Library
+import XCTest
+
+final class NumberFormatterTests: TestCase {
```

Comment:
These tests should only test the attributed string (the format is tested in FormatTests)